homeassistant/components/voip/assist_satellite.py

```diff
@@ -354,7 +406,11 @@ async def stt_stream():
                 # length of the TTS audio.
                 await self._tts_done.wait()
         except TimeoutError:
+            # This shouldn't happen anymore, we are detecting hang ups with a separate task
+            _LOGGER.exception("Timeout error")
             self.disconnect()  # caller hung up
+        except asyncio.CancelledError:
+            _LOGGER.debug("Pipeline cancelled")
```

Comment:
Should I re-raise the CancelledError here if the current task is cancelling as well?

Reply:
Yes